HistoryEntityManager/src/main/resources/application.properties

```diff
@@ -11,11 +10,14 @@ mysettings.postgres.username=${dbuser:ngb}
 mysettings.postgres.password=${dbpass:ngb}
 mysettings.postgres.database-name=${dbname:ngb}
 mysettings.gateway.host=${gateway.host:localhost}
-mysettings.gateway.port=${gateway.port:9090}
-atcontext.url=http://${mysettings.gateway.host}:${mysettings.gateway.port}/ngsi-ld/contextes/
+mysettings.gateway.port=${gateway.port:1040}
```

Comment:
please change this back the gateway port is not the same as the http port of the microservice. this only is the case in the aio but for the dist microservice settings they are not the same. and they are logically not the same. the gateway port and host is for user to be reached so any info we give out should point to gateway and any internal calling to at-context should point to at-context-server